plugins/armorblox/icon_armorblox/util/constants.py

```diff
@@ -0,0 +1,3 @@
+ARMORBLOX_INCIDENT_API_TIME_FORMAT = "%Y-%m-%dT%H:%M:%SZ"
+ARMORBLOX_INCIDENT_API_TIME_DELTA_IN_DAYS = 1
+DEFAULT_INTERVAL_VALUE = 1
```

Comment:
Please always set that value the same as `default` in plugin.spec.yaml file 😄 
```suggestion
DEFAULT_INTERVAL_VALUE = 600
```